### DISCUSSION TITLE
Budibase Cloud - Server IP Addresses for Whitelisting MySQL Connection

### DISCUSSION THREAD

**Discussion by mikedownesdev** [Budibase/budibase#3114] [Help] [answered]
In order to connect to my MySQL instance, I need to authorize networks that will attempting connection. Right now I'm using the Cloud beta and was wondering if the IP ranges of the Budibase servers were available. 

For some context, I was able to connect to my instance from a similar product, Appsmith, by authorizing the networks that they note in this documentation page -- https://docs.appsmith.com/core-concepts/connecting-to-data-sources

**Comment by shogunpurple** [comment, marked as answer]
Hi @michaelfdownes, thanks for raising. The IP address you need to whitelist for our production cluster is:

`34.246.43.190`

I'll update the documentation to reflect this also.
